### ISSUE TITLE
Mechanic to track time of users 

### ISSUE THREAD

**Issue by ren-kun** [StarryPy/StarryPy3k#85]
Would it be possible to add a time card mechanic?

Say 1 hour of game stores 1 value to a text file for a single IP. storing

IP - Name - Hours on -> Dump to file -> shuffle logs after 1 week

**Comment by medeor413** [comment]
Can't reproduce this. I'm gonna close it for now, feel free to reopen it if it happens again.

**Comment by medeor413** [comment]
Whoops, wrong issue. So, if I'm understanding correctly, you want to see how long a player has spent online in a given timeframe?

**Comment by ren-kun** [comment]
Correct. It can be used to reward players for how long they have been playing in a given amount of time.
